### DISCUSSION TITLE
I am trying to run enable3D basic setup, I keep getting nothing

### DISCUSSION THREAD

**Discussion by Igglepud** [enable3d/enable3d#286] [Help / Questions]
When I run the Basic Setup code, all I get is a black screen. Console log shows a Scene3D and the alert triggers. No errors present, but again, all I see is a black screen.

let MainScene = new ENABLE3D.Scene3D("MainScene");

MainScene.init = function () {
  this.accessThirdDimension();
};

MainScene.preload = function () {};

MainScene.create = function () {
  console.log(this);

  this.third.warpSpeed();

  // add a simple box
  this.third.physics.add.box();
  alert();
};

**Comment by vitalspace** [comment]
If you plan to use functions instead of classes, you should check out this code.

https://github.com/enable3d/enable3d-website/blob/master/src/examples/native-three-with-physics.html

**Reply by vitalspace** [reply]
with classes 

https://github.com/enable3d/enable3d-website/blob/master/src/examples/standalone-mode.html

**Reply by Igglepud** [reply]
Will those still work with Phaser? That's my biggest concern.

**Reply by Igglepud** [reply]
Well, that and avoiding webpack.

**Reply by vitalspace** [reply]
> Will those still work with Phaser? That's my biggest concern.

for phaser

https://github.com/enable3d/enable3d-website/blob/master/src/examples/simple-setup.html

**Reply by vitalspace** [reply]
Fortunately for the phaser examples you just have to add "third". like "this.third.physics.add.box()".

**Comment by Igglepud** [comment]
Yeah, simple setup is what I'm using.

**Comment by Igglepud** [comment]
This is what I get for not using the template.
